lib/galaxy/model/migrate/versions/0161_worker_process_pid.py

```diff
@@ -0,0 +1,38 @@
+"""
+Adds `pid` column to worker_process table.
+"""
+
+from __future__ import print_function
+
+import logging
+
+from sqlalchemy import (
+    Column,
+    MetaData
+)
+
+from galaxy.model.custom_types import UUIDType
```

Comment:
This isn't used, but you need to import the sqlachemy Integer type and use that instead of the python int.
```suggestion
```